src/actions.ts

```diff
@@ -0,0 +1,43 @@
+// Sample set of actions for authentication. These are the necessary actions in order to make the Auth flow functional in your app.
+interface Form {
+    email: string;
+    username: string;
+    password: string;
+    newsletter: Boolean;
+}
+
+const Actions = () => {
```

Comment:
This could be an interface or an abstract class. Makes more sense from the outside